### PR TITLE
Added TestClasses and DBTest.

### PR DESCRIPTION
**Pull request by satkab**
Added TestClasses folder in the 6220_2020 team folder. Inside of it added DBTest class for driver bot practice code.

### CODE REVIEW

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/DBTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        // on a Core Device Interface Module, configured to be a sensor of type "AdaFruit IMU",
+        // and named "imu". Certain parameters must be specified before using the imu.
+        BNO055IMU.Parameters parameters = new BNO055IMU.Parameters();
+        parameters.angleUnit = BNO055IMU.AngleUnit.DEGREES;
+        parameters.accelUnit = BNO055IMU.AccelUnit.METERS_PERSEC_PERSEC;
+        parameters.calibrationDataFile = "AdafruitIMUCalibration.json";
+        parameters.loggingEnabled = true;
+        parameters.loggingTag = "IMU";
+        imu = hardwareMap.get(BNO055IMU.class, "imu");
+        imu.initialize(parameters);
```
Comment (Prime-235711) [inline Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/DBTest.java:46]: Do you know what the imu does?

**Review by malavika1 — APPROVED** [review]

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/DBTest.java (hunk 1 above)
Comment (satkab) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/DBTest.java:46]: No

**Review by SwerveRoboticsTeams — COMMENTED** [review]
What does this class do?  Could you add some comments in the code regarding the purpose of this class?

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/DBTest.java (hunk 1 above)
Comment (satkab) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/DBTest.java:46]: Now I do :)

Comment (Prime-235711) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/DBTest.java:46]: It has a gyroscope, accelerometer, etc.

**Review by Prime-235711 — APPROVED** [review]

**Pull request merged**